.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -26,7 +26,7 @@
 #       uses: kyma-project/eventing-tools/.github/workflows/bump-sec-scanners-config-reusable.yml@main
 #       with:
 #         version_tag: 2.3.4
-#         timeout: 3600 # 1 hour
+#         TIMEOUT: 3600 # 1 hour
```

Comment:
Lowercase or Uppercase `version_tag` and `TIMEOUT`, please keep this consistent.